Add User Feedback section for Rails (#5436)
getsentry/sentry-docs: src/includes/enriching-events/user-feedback/example-widget/ruby.rails.mdx

> In Rails, being able to serve dynamic (e.g. `.html.erb`) pages in response to errors is required in order to pass the needed `event_id` to the JavaScript SDK. Normally, this looks like overwriting the `config.exceptions_app` setting to point at your own router, and then calling your own controller to handle the error. Several tutorials or gems exist to allow for this.

Reviewer: I'm not clear on what you mean in the last sentence here. Do you mean there are resources available to teach people how to do what you're describing? And are you using "gems" in a colloquial way or is there some other meaning here?

```suggestion
In Rails, being able to serve dynamic (for example, `.html.erb`) pages in response to errors is required to pass the needed `event_id` to the JavaScript SDK. Typically, this is done by overwriting the `config.exceptions_app` setting to point to your own router, and then calling your own controller to handle the error. Several tutorials or gems exist to allow for this.
```

Author: "gem" is the term for a Ruby library or package.
What I'm trying to say here is that people should consult external resources, whether that's following a tutorial to do it themselves, or installing a gem to handle this. This is in part because the details are hairy, and partly because I don't think these docs should be opinionated about how to do this.
There's definitely a less-awkward way of wording it. Maybe, "More detailed instructions on this configuration are available online. Alternatively, you can install a gem to handle this for you."

Reviewer: Thanks for the clarification. I've gone ahead and updated my suggestion.

> Once you are able to serve dynamic exception pages, supporting user feedback is straightforward.

Reviewer: We try to avoid telling people what their experience of carrying out a task will be in case it doesn't end up that way for them :-)

```suggestion
Once you're able to serve dynamic exception pages, you can support user feedback.
```

Make sure you've got the JavaScript SDK available:

```ERB
<script
  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.min.js"
  integrity="sha384-{{ packages.checksum('sentry.javascript.browser', 'bundle.min.js', 'sha384-base64') }}"
  crossorigin="anonymous"
></script>
```

> And the template that brings up the dialog:

Reviewer: ```suggestion
Additionally, you need the template that brings up the dialog:
```
